### ISSUE TITLE
Emoji double-check confirmation before moderation actions.

### ISSUE THREAD

**Issue by ShadowJonathan** [matrix-org/mjolnir#375]
#### Is your feature request related to a problem? Please describe.
When operating mjolnir, and using text-based commands, its often easy to mistype or make mistakes, which then have to be corrected after the fact.

These mistakes can range from a single typo ("sapm") to destructive actions (banning `@spambot* :matrix.org`. where the bot interprets `:matrix.org` as the "reason")

#### Describe the solution you'd like
Have an opt-in configuration where the bot repeats the command back to you in plain english, "You want me to ban `@abcd:efgh`, who is currently in `8` protected rooms?", and then allow the user to press ✅ or ❌ on that message, to confirm or deny the command.

**Comment by turt2live** [comment]
Non-glob bans are recoverable if they go sideways, but globs are harder to do. Mjolnir has a config option these days to require a `--force` flag with globs, making the sender accountable for what they're sending.
